.github/workflows/data-puller.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         run: |
           python scripts/update_data.py
           python scripts/process_for_site.py
+          python scripts/metrics/social/automate_sprout.py
```

Comment:
Will this work if there are no working files? Maybe we can't automate this at the moment.